test/functional/mining_basic.py

```diff
@@ -83,20 +81,8 @@ def assert_submitblock(block, result_str_1, result_str_2=None):
         self.log.info("getblocktemplate: Test capability advertised")
         assert 'proposal' in tmpl['capabilities']
         assert 'coinbasetxn' not in tmpl
-
-        next_height = int(tmpl["height"])
-        coinbase_tx = create_coinbase(height=next_height)
-        # sequence numbers must not be max for nLockTime to have effect
-        coinbase_tx.vin[0].nSequence = 2 ** 32 - 2
-        coinbase_tx.rehash()
-
-        # round-trip the encoded bip34 block height commitment
-        assert_equal(CScriptNum.decode(coinbase_tx.vin[0].scriptSig), next_height)
```

Comment:
I think this one was actually testing the miner, so should be kept in this file.

Reply:
I don't understand. How is this testing the miner? It's asserting on the return value of `create_coinbase()`, which is purely a python helper function.